pages/Pressure-Chamber.md

```diff
@@ -0,0 +1,13 @@
+The Pressure Chamber is in the [Basic Machines](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machines) category. It is a multiblock structure.<br>
+
+## Obtaining
+The Pressure Chamber is a multiblock structure, it consists of a dispenser, 2 smooth stone slabs, a piece of glass, a cauldron and 4 pistons.<br>
+
+A complete setup should look like this:
+
+![Pressure Chamber Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-pressure-chamber.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your items. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Pressure Chamber is used to put pressure on items.
```

Comment:
This is a wiki, please be more precise than that.
State that it is used to craft various items and that they should look into the Slimefun Guide for more.